__tests__/topics.test.js

```diff
@@ -209,3 +209,62 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  test("POST: 201, responds with an object containing a comment key and value of an object", () => {
+    const comment = { username: "icellusedkars", body: "cool comment" };
+    return supertest(app)
+      .post("/api/articles/9/comments")
+      .send(comment)
+      .expect(201)
+      .then(({ body }) => {
+        console.log(body);
+        expect(Object.keys(body)[0]).toBe("comment");
+        expect(body.comment).toBeInstanceOf(Object);
+      });
+  });
+  test("POST: 201, responds with an object of comment with the correct keys", () => {
+    const comment = { username: "icellusedkars", body: "cool comment" };
+    return supertest(app)
+      .post("/api/articles/9/comments")
+      .send(comment)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment).toMatchObject({
+          comment_id: expect.any(Number),
+          body: expect.any(String),
+          article_id: expect.any(Number),
+          author: expect.any(String),
+          votes: expect.any(Number),
+          created_at: expect.any(String),
+        });
+      });
+  });
+  test("POST: 201, responds with the requested comment", () => {
+    const comment = { username: "icellusedkars", body: "cool comment" };
+    return supertest(app)
+      .post("/api/articles/9/comments")
+      .send(comment)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment).toMatchObject({
+          comment_id: 19,
+          body: "cool comment",
+          article_id: 9,
+          author: "icellusedkars",
+          votes: 0,
+          created_at: expect.any(String),
+        });
+      });
+  });
+  test("GET 400: responds with 400 if request is NaN", () => {
+    const comment = { username: "icellusedkars", body: "cool comment" };
+    return supertest(app)
+      .post("/api/articles/NaN/comments")
+      .send(comment)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body).toEqual({ msg: "Bad Request" });
+      });
+  });
```

Comment:
A few tests to add, please:

- [ ] 404 - article not found
- [ ] 404 - username not found
- [ ] 400 missing required field on the post body object, ie its empty or missing the body key